README.md

```diff
@@ -91,7 +91,8 @@ DWN SDK includes a polyfilled distribution that can imported in a `module` scrip
     const dwn = await Dwn.create({ messageStore, dataStore, eventLog });
 
     // Generate a did:key DID (Decentralized Identifier).
-    const didKey = await DidKeyResolver.generate();
+    // TODO:
```

Comment:
missing TODO?

Reply:
Thanks, forgot to remove the self-reminder after fixing!